src/main/scala/io/iohk/ethereum/blockchain/data/GenesisDataLoader.scala

```diff
@@ -137,7 +137,8 @@ class GenesisDataLoader(
       unixTimestamp = BigInt(genesisData.timestamp.replace("0x", ""), 16).toLong,
       extraData = genesisData.extraData,
       mixHash = genesisData.mixHash.getOrElse(zeros(hashLength)),
-      nonce = genesisData.nonce)
+      nonce = genesisData.nonce,
+      slotNumber = 0 /* TODO: Check if this have sense */)
```

Comment:
Taking into account that the block number is configured in a similar way, maybe it's okay to have it configured this way and we should remove this TODO?

Reply:
:+1: